v23.1/known-limitations.md

```diff
@@ -30,17 +30,59 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
     [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/84680)
 
-### No support for placeholders in `AS OF SYSTEM TIME`
+### Limitations for `SELECT FOR UPDATE`
 
-CockroachDB does not support placeholders in [`AS OF SYSTEM TIME`](as-of-system-time.html). This means that the time value has to be embedded in the SQL string.
+- [`SELECT FOR UPDATE`](select-for-update.html) places locks on each key scanned by the base index scan. This means that even if some of those keys are later filtered out by a predicate which could not be pushed into the scan, they will still be locked.
 
-[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/30955)
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/75457)
+
+- [`SELECT FOR UPDATE`](select-for-update.html) only places an unreplicated lock on the index being scanned by the query. This diverges from PostgreSQL, which aquires a lock on all indexes.
+
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/57031)
+
+### Limitations for composite types
+
+- Avro [changefeeds](create-and-configure-changefeeds.html) do not support user-defined composite (tuple) types.
+
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/102903)
+
+- Parquet [changefeeds](create-and-configure-changefeeds.html) do not support user-defined composite (tuple) types.
```

Comment:
Yes, this can be removed from here.

Reply:
Oh sorry @lnhsingh I just saw this was merged. I am working on an issue related to this right now, I can remove!

Reply:
Thank you!